code/__HELPERS/logging/attack.dm

```diff
@@ -80,5 +80,5 @@
 
 	GLOB.bombers += bomb_message
 
-	if(message_admins)
+	if(message_admins && !istype(bomb.loc, /area/deathmatch)) // Don't spam the logs with deathmatch bombs
```

Comment:
Using get_area might be a bit safer for the .loc calls. It's more thorough and less prone to returning non-area types.